terraform/modules/tf_stream_alert/main.tf

```diff
@@ -31,7 +32,7 @@ resource "aws_lambda_alias" "rule_processor_production" {
   name             = "production"
   description      = "Production StreamAlert Rule Processor Alias"
   function_name    = "${aws_lambda_function.streamalert_rule_processor.arn}"
-  function_version = "${var.rule_processor_version}"
+  function_version = "${aws_lambda_function.streamalert_rule_processor.version}"
```

Comment:
This interpolation enables the rollbacks to work properly right? I'm assuming Terraform refreshes state after the boto3 call is made

Reply:
Yep! After a rollback, a `terraform apply` would restore the aliases to the latest published version (as long as `-refresh` was enabled, which it is by default and in all of our CLI). A deploy after a rollback would first publish a new version and then skip ahead from the rollback to the most recent (jumping forward 2 versions).

I just tested both of these exact scenarios and confirm they work as expected